CHANGELOG.md

```diff
@@ -1,6 +1,36 @@
 Changelog
 =========
 
+2.0.1
+-----
+
+### Overview of changes
+- Update bundled certificates as of 2022-02-01. [#670]
+- Bug fix: Hook priority should be respected. [#452], [#647]
+- Docs: the Hook documentation has been updated to reflect the current available hooks. [#646]
+- General housekeeping. [#635], [#649], [#650], [#653], [#655], [#658], [#660], [#661], [#662], [#669], [#671], [#672], [#674]
+
+Props [@costdev][gh-costdev], [@jegrandet][gh-jegrandet] [@jrfnl][gh-jrfnl], [@schlessera][gh-schlessera]
```

Comment:
```suggestion
Props [@alpipego][gh-alpipego], [@costdev][gh-costdev], [@jegrandet][gh-jegrandet] [@jrfnl][gh-jrfnl], [@schlessera][gh-schlessera]
```

Reply:
Alex should be added as well for this release, as he did code review. However, I can't "suggest" to add the shortcut below.